docs/res

```diff
@@ -0,0 +1,1996 @@
+Apache Maven 3.6.1 (d66c9c0b3152b2e69ee9bac180bb8fcc8e6af555; 2019-04-04T21:00:29+02:00)
```

Comment:
This file looks like an unintended inclusion.

Reply:
Thanks, removed.